### DISCUSSION TITLE
Visual Studio Code doesn’t see .cpp files

### DISCUSSION THREAD

**Discussion by gospodryad** [microsoft/vscode-cpptools#12231] [General]
getting a message «undefined reference to»

**Comment by gospodryad** [comment]
here is my tasks.json file (tried to fix this by using `"${workspaceFolder}/*.cpp"` instead of `"${file}"`, but ended up with the same problem)
`{
    "tasks": [
        {
            "type": "cppbuild",
            "label": "C/C++: g++ сборка активного файла",
            "command": "/usr/bin/g++",
            "args": [
                "-fdiagnostics-color=always",
                "-g",
                "${workspaceFolder}/*.cpp",
                "-o",
                "${fileDirname}/${fileBasenameNoExtension}"
            ],
            "options": {
                "cwd": "${fileDirname}"
            },
            "problemMatcher": [
                "$gcc"
            ],
            "group": {
                "kind": "build",
                "isDefault": true
            },
            "detail": "Задача создана отладчиком."
        }
    ],
    "version": "2.0.0"
}`

**Reply by sean-mcmanus** [reply]
@devilnute Have you tried `"*.cpp"`?
